src/app/components/MediaLoader/types.ts

```diff
@@ -23,6 +23,7 @@ export type PlayerConfig = {
     holdingImageURL: string;
     items: PlaylistItem[];
     guidance?: string;
+    embedRights?: 'allowed';
```

Comment:
**Note for reviewers:** this has been moved from the `PlaylistItem` -> `PlaylistObject` as per the SMP Cookbook